config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/tasks/zip/builder/JdbcConnectionEntityBuilder.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2018 CA. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+package com.ca.apim.gateway.cagatewayconfig.tasks.zip.builder;
+
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.Bundle;
+import com.ca.apim.gateway.cagatewayconfig.tasks.zip.beans.JdbcConnection;
+import com.ca.apim.gateway.cagatewayconfig.util.IdGenerator;
+import com.ca.apim.gateway.cagatewayconfig.util.entity.EntityTypes;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildAndAppendPropertiesElement;
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BuilderUtils.buildPropertiesElement;
+import static com.ca.apim.gateway.cagatewayconfig.util.gateway.BundleElementNames.*;
+import static com.ca.apim.gateway.cagatewayconfig.util.xml.DocumentUtils.*;
+
+public class JdbcConnectionEntityBuilder implements EntityBuilder {
+
+    private static final String STORED_PASSWORD_REF_FORMAT = "${secpass.%s.plaintext}";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'PASSWORD' detected in this expression, review this potentially hard-coded credential. ([squid:S2068](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=53&issues=AWXznoBUOlZ5QoPzBphu&open=AWXznoBUOlZ5QoPzBphu)

[ISSUE_KEY]: AWXznoBUOlZ5QoPzBphu